posts/2023-10-05-Rust-1.73.0.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Announcing Rust 1.73.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.73.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, you can get 1.73.0 with:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.73.0](https://github.com/rust-lang/rust/releases/tag/1.73.0) on GitHub.
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.73.0 stable
+
+## Cleaner panic messages
```

Comment:
@m-ou-se I think I replicated this correctly from your comment (https://github.com/rust-lang/rust/pull/112849#issuecomment-1678731335) -- thanks for writing this up.

It does look a little odd to me with the dark backgrounds in contrast with the code background, but I think that's OK.

<img width="613" alt="image" src="https://github.com/rust-lang/blog.rust-lang.org/assets/5047365/986db5ce-6691-451d-bf41-5575ae9030c7">

Reply:
Why use manual styling at all, versus markdown code blocks?